articles/quickstart/native/android/_includes/_login.md

```diff
@@ -2,6 +2,9 @@
 
 In our login method we create a new `Auth0` instance to hold the credentials. Then by using the `WebAuthProvider` class we can authenticate with any connection enabled for our client in the Auth0 dashboard. We also tell the provider to use the custom scheme `demo` to construct the expected **Callback URL**.
 
+Additionally, to make the server return Open ID Connect compliant responses we must either request an `audience` or flip the OIDC Conformant switch on in the client's dashboard Advanced OAuth settings. You can read more about that [here](https://auth0.com/docs/api-auth/intro#how-to-use-the-new-flows).
```

Comment:
To ensure an Open ID Connect compliant responses you can either request an audience or enable OIDC Conformance in your dashboard under Advanced OAuth settings. You can read more about that here.